### DISCUSSION TITLE
Weird highlighted code labels.

### DISCUSSION THREAD

**Discussion by JoseManuelValderrama73** [LazyVim/LazyVim#4562] [Q&A] [answered]
LazyVim is working perfectly, but today when I started it, my code was full of type labels, which are unreadable because of the highlight. I think I didn't do anything to enable them, they just appeared on their own. Does someone know how to disable them?
![Screenshot 2024-10-09 at 20 24 17](https://github.com/user-attachments/assets/d333e333-9388-4654-82db-68c1356c9253)

**Comment by dpetka2001** [comment, marked as answer]
I think these are inlay hints and they are enabled by default on Neovim >=0.10. The way they appear is just how your colorscheme sets the highlight groups to be. You want to check `LspInlayHint` highlight group name what it is set to in your colorscheme. Make appropriate changes to another color of your liking.

Alternatively, you can use `<leader>uh` to temporarily toggle them on/off during your running Neovim session or for a permanent solution you can disable them with the following
```lua
return {
  {
  "neovim/nvim-lspconfig",
  opts = function(_, opts)
    opts.inlay_hints = {
      enabled = false,
    }
  end,
  },
}
```

**Reply by JoseManuelValderrama73** [reply]
That's it, thank you SOO much!
